core/src/test/java/org/infinispan/statetransfer/RemoteGetDuringStateTransferTest.java

```diff
@@ -54,40 +63,88 @@
+   | 132_22 | 6.1 | T1            | 1:T3              | T2                 | Y*    | 2:T2              | T2/T3/T4           |
+   | [4]    |     | T1            | 1:T3              | T2                 | Y*    | 2:T3              | T0/T1              |
+   | 132_32 | 6.2 | T1            | 1:T3              | T2                 | Y*    | 2:T3              | T2/T3/T4           |
+   | [2]    |     | T1            | 1:T3              | T2                 | Y*    | 2:T4              | T0/T1/T2           |
+   | [1]    |     | T1            | 1:T3              | T2                 | Y*    | 2:T4              | T3/T4              |
+   | [2]    |     | T1            | 1:T3              | T3                 | Y     | 2:T0/T1           |                    |
+   | [4]    |     | T1            | 1:T3              | T3                 | Y     | 2:T2              | T0/T1/T2           |
+   | 133_23 | 7.1 | T1            | 1:T3              | T3                 | Y     | 2:T2              | T3/T4              |
+   | [4]    |     | T1            | 1:T3              | T3                 | Y     | 2:T3              | T0/T1/T2           |
+   | 133_33 | 7.2 | T1            | 1:T3              | T3                 | Y     | 2:T3              | T3/T4              |
+   | [4]    |     | T1            | 1:T3              | T3                 | Y     | 2:T4              | T0/T1/T2           |
+   | [1]    |     | T1            | 1:T3              | T3                 | Y     | 2:T4              | T3/T4              |
+   | [2]    |     | T1            | 1:T3              | T4                 | Y     | 2:T0/T1/T2        |                    |
+   | [1]    |     | T1            | 1:T3              | T4                 | Y     | 2:T3/T4           | T4                 |
+   | [4]    |     | T1            | 1:T4              | T0/T1/T2           |       |                   |                    |
+   | [2]    |     | T1            | 1:T4              | T3/T4              | Y     | 2:T0/T1/T2        |                    |
+   | [4]    |     | T1            | 1:T4              | T3/T4              | Y     | 2:T3/T4           | T0/T1/T2           |
+   | [1]    |     | T1            | 1:T4              | T3/T4              | Y     | 2:T3/T4           | T3/T4              |
+   | [2]    |     | T2            | 1:T0              |                    |       |                   |                    |
+   | [4]    |     | T2            | 1: *, 2:  *       | T0/T1              |       |                   |                    |
+   | 2112   | 8.1 | T2            | 1:T1, 2: T1       | T2/T3/T4           | N1    |                   |                    |
+   | 2122   | 8.2 | T2            | 1:T1, 2: T2       | T2/T3/T4           | N1+2  |                   |                    |
+   | 2132   | 8.3 | T2            | 1:T1, 2: T3/T4    | T2/T3/T4           | N1+2  |                   |                    |
```

Comment:
It's not the final 2, it's the second number which got duplicated. If you read the topology references as written on the line in the table, you'll see those numbers:
2132: Originator in topology *2* sent request to both two nodes, node 1 receiving that in topology *1* and node 2 receiving that in topology *3*. The response i received still in topology *2*.